config/locales/en.yml

```diff
@@ -77,6 +77,9 @@ en:
             invalid_type_parameter:
               one: "%{invalid_types} is an invalid parameter type."
               other: "%{invalid_types} are invalid parameter types."
+            unacceptable_boolean: "%{field} must be either true or false without quotes."
```

Comment:
"without quotes" seems like pretty indirect advice to me. Could we just say it needs to be a boolean, and assume that app developers can tell booleans from strings?